gui/src/App.tsx

```diff
@@ -142,11 +143,13 @@ export default function App() {
   const [updateFound, setUpdateFound] = useState('');
   useEffect(() => {
     async function fetchReleases() {
-      const releases: Release[] = await fetch(
+      const releases = await fetch(
         `https://api.github.com/repos/${GH_REPO}/releases`
-      ).then((res) => res.json());
+      )
+        .then((res) => res.json())
+        .then((json: any[]) => json.filter((rl) => rl?.prerelease === false));
```

Comment:
no.

Reply:
its better than bringing a whole dependency for one type check